addrmgr/addrmanager_test.go

```diff
@@ -910,148 +913,159 @@ func TestValidatePeerNa(t *testing.T) {
-		valid:         false,
-		reach:         Default,
-	}, {
-		name:          "rfc4380 to rfc4380",
-		localAddress:  rfc4380IPAddress,
-		remoteAddress: rfc4380IPAddress,
-		valid:         true,
-		reach:         Teredo,
-	}, {
-		name:          "unroutable ipv4 to rfc4380",
-		localAddress:  unroutableIpv4Address,
-		remoteAddress: rfc4380IPAddress,
-		valid:         false,
-		reach:         Default,
-	}, {
-		name:          "routable ipv4 to rfc4380",
-		localAddress:  routableIpv4Address,
-		remoteAddress: rfc4380IPAddress,
-		valid:         true,
-		reach:         Ipv4,
-	}, {
-		name:          "routable ipv6 to rfc4380",
-		localAddress:  routableIpv6Address,
-		remoteAddress: rfc4380IPAddress,
-		valid:         true,
-		reach:         Ipv6Weak,
-	}, {
-		name:          "routable ipv4 to routable ipv4",
-		localAddress:  routableIpv4Address,
-		remoteAddress: routableIpv4Address,
-		valid:         true,
-		reach:         Ipv4,
-	}, {
-		name:          "routable ipv6 to routable ipv4",
-		localAddress:  routableIpv6Address,
-		remoteAddress: routableIpv4Address,
-		valid:         false,
-		reach:         Unreachable,
-	}, {
-		name:          "unroutable ipv4 to routable ipv6",
-		localAddress:  unroutableIpv4Address,
-		remoteAddress: routableIpv6Address,
-		valid:         false,
-		reach:         Default,
-	}, {
-		name:          "unroutable ipv6 to routable ipv6",
-		localAddress:  unroutableIpv6Address,
-		remoteAddress: routableIpv6Address,
-		valid:         false,
-		reach:         Default,
-	}, {
-		name:          "unroutable ipv4 to routable ipv6",
-		localAddress:  unroutableIpv4Address,
-		remoteAddress: routableIpv6Address,
-		valid:         false,
-		reach:         Default,
-	}, {
-		name:          "routable ipv4 to unroutable ipv6",
-		localAddress:  routableIpv4Address,
-		remoteAddress: unroutableIpv6Address,
-		valid:         false,
-		reach:         Unreachable,
-	}, {
-		name:          "routable ivp6 rfc4380 to routable ipv6",
-		localAddress:  rfc4380IPAddress,
-		remoteAddress: routableIpv6Address,
-		valid:         true,
-		reach:         Teredo,
-	}, {
-		name:          "routable ipv4 to routable ipv6",
-		localAddress:  routableIpv4Address,
-		remoteAddress: routableIpv6Address,
-		valid:         true,
-		reach:         Ipv4,
-	}, {
-		name:          "tunnelled ipv6 rfc3964 to routable ipv6",
-		localAddress:  rfc3964IPAddress,
-		remoteAddress: routableIpv6Address,
-		valid:         true,
-		reach:         Ipv6Weak,
-	}, {
-		name:          "tunnelled ipv6 rfc6052 to routable ipv6",
-		localAddress:  rfc6052IPAddress,
-		remoteAddress: routableIpv6Address,
-		valid:         true,
-		reach:         Ipv6Weak,
-	}, {
-		name:          "tunnelled ipv6 rfc6145 to routable ipv6",
-		localAddress:  rfc6145IPAddress,
-		remoteAddress: routableIpv6Address,
-		valid:         true,
-		reach:         Ipv6Weak,
-	}}
+		localAddr     string
+		remoteAddr    string
+		expectedBool  bool
+		expectedReach NetAddressReach
+	}{
+		{
```

Comment:
Please stick with the more compact style we use everywhere else.

```
}{{
	...
}, {
	...
}}
```

Reply:
Fixed.